src/api/observationAPI.ts

```diff
@@ -14,18 +14,18 @@ export interface ReadingValue {
   fwi: number | null
 }
 
-export interface Reading {
+export interface Observation {
   station: Station
-  values: ReadingValue[]
+  values: ObservedValue[]
 }
 
-export interface ReadingsResponse {
-  hourlies: Reading[]
+export interface ObservationsResponse {
+  hourlies: Observation[]
 }
 
-export async function getReadings(stationCodes: number[]): Promise<Reading[]> {
+export async function getObservations(stationCodes: number[]): Promise<Observation[]> {
```

Comment:
I've added onto the api re-factor ticket to confirm with this: https://app.zenhub.com/workspaces/wildfire-predictive-services-5e321393e038fba5bbe203b8/issues/bcgov/wps/268